### ISSUE TITLE
Possible to have separate profiles per class?

### ISSUE THREAD

**Issue by matthewhively** [rgd87/NugEnergy#11]
I would like to have the energy tick timer centrally located, but the mana regen timer off to the side somewhere. Is this possible somehow?

**Comment by oRauu** [comment]
Yes please, i have different position for 2 classes, kinda hard to optimize it
